CHANGELOG.adoc

```diff
@@ -12,6 +12,15 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
 
 - Include delete with entity parameter on Template
 
+== [1.1.0] - 2025-09-14
```

Comment:
We did not release a new version; thus, please keep it on the `Unreleased` as well.